controller/src/main/java/uni/miskolc/ips/ilona/measurement/controller/MeasurementController.java

```diff
@@ -131,17 +119,7 @@ public final boolean recordMeasurement(@RequestBody final MeasurementRegistratio
         if (measurementRegistrationRequest.getRfidtags() != null) {
             measurement.setRfidtags(new RFIDTags(new HashSet<byte[]>(measurementRegistrationRequest.getRfidtags().getRfidTag())));
```

Comment:
In previous if-s, the creation of the instance is not the parameter of a setter.

Reply:
instantiation has been moved before the setter method